documentation/user_kit.md

```diff
@@ -261,6 +262,75 @@ Partners can revoke user access to given doors. This will be done by using a par
 
         ⇒ Contact Latch Support
 
+### Get user
+
+Partners can fetch a single user. This will be done by using a partner-scoped token from the BE.
+
+1. GET user request from the Partner BE to the Latch BE
+
+    ```
+    GET https://rest.latchaccess.com/access/sdk/v1/users/:user
```

Comment:
Consider adding some clarity on what :user is. It's the user uuid as can be seen from the users endpoint